### ISSUE TITLE
Add documentation about the PhpSpec opinions

### ISSUE THREAD

**Issue by stof** [phpspec/phpspec#635]
PhpSpec is opinionated about preactices it wants to promote. The doc should have a chapter (or maybe a whole section with several chapters) describing these practices.

This has been suggested in the discussion in https://github.com/phpspec/phpspec/issues/632

**Comment by rtuin** [comment]
Agreed. Of course the current phpspec code display these opinions, but is there a list of them somewhere to start off with?

**Comment by ciaranmcnulty** [comment]
Mostly here https://github.com/phpspec/phpspec/blob/master/docs/cookbook/configuration.rst

**Comment by ciaranmcnulty** [comment]
I think it makes sense like @stof suggests to have somewhere talking about best practices, not just limited to options.

Looking at the options page though, it could do with some love.

**Comment by stof** [comment]
@ciaranmcnulty most PhpSpec opinions are precisley things which *don't* have an option.

**Comment by stof** [comment]
And it is a totally different topic than the configuration page

**Comment by yguedidi** [comment]
:+1: this can also help newcomers with best practices

**Comment by ciaranmcnulty** [comment]
Hah I consistently misread opinions as options through this conversation :-)
